packages/frameworks/src/frameworks.ts

```diff
@@ -1932,6 +1932,30 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'Storybook',
```

Comment:
Correct.

Interestingly enough, we used to have Storybook in the list of frameworks, but it was removed (https://github.com/vercel/vercel/pull/3864) because it didn't have a detector.

Reply:
@cb1kenobi Lets add a comment here to explain that is intentional so it doesn't get removed in the future

Reply:
I'm also curious - why not have any detectors?

Reply:
I added the detector. Originally I didn't think it mattered since according to https://storybook.js.org/docs/react/get-started/install:

> Storybook needs to be installed into a project that is already set up with a framework.

In theory, we will detect the site as Next.js or whatever before we would detect it as Storybook.

However, maybe the user is doing something special like the UI components are in an external package or something which would cause the project to be detected as Storybook.